components/Reformeur.jsx

```diff
@@ -345,6 +410,11 @@ class Reformeur extends Component {
             this.UpdateDecote(whichdecote, newvalue)
             // success=true;
         }
+        if (name.substring(0, 10) == "plafond_qf") {
+            const whichplaf = name.substring(10)
+            this.UpdatePlafond(whichplaf, newvalue)
+            // success=true;
+        }
         const bodyreq = this.cas_types_defaut
             ? JSON.stringify({
```

Comment:
Hors PR mais
plutot que de répére JSON.stringify et reforme: this.state.reforme
on peut écrire
```
const extend = this.cas_types_defaut
  ? { deciles: false }
  : { description_cas_types: this.state.cas_types }
JSON.stringify({
  reforme: this.state.reforme,
  ...extend
})
```